src/dsp/BandSplitter/BandSplitterProcessor.h

```diff
@@ -38,29 +39,55 @@ struct Params : chowdsp::ParamHolder
         juce::ParameterID { "band_split_cutoff2", ParameterVersionHints::version1_0_0 },
         "Band Splitter Cutoff 2",
         chowdsp::ParamUtils::createNormalisableRange (20.0f, 20000.0f, 2000.0f),
-        10000.0f
+        6000.0f
     };
 
+    //for use when we have 4 bands
+    //    chowdsp::FreqHzParameter::Ptr cutoff3{
+    //        juce::ParameterID { "band_split_cutoff3", ParameterVersionHints::version1_0_0 },
+    //        "Band Splitter Cutoff 3",
+    //        chowdsp::ParamUtils::createNormalisableRange (20.0f, 20000.0f, 2000.0f),
+    //        10000.0f
+    //    };
+
+    //change from bool to choice parameter when adding a 4th band
```

Comment:
Hmm... we might need to be careful with this. Changing the parameter type could end up breaking previously saved sessions.